docs/reference/conditions-reference.md

```diff
@@ -272,19 +276,31 @@ The supported map parameters are:
     </tr>
     <tr>
       <td> default_range </td>
-      <td>  
+      <td>
         Configures the commit range that is examined on the default branch. The
         default value is <b>$SEMAPHORE_GIT_COMMIT_RANGE</b> which behavior is
         described above. It accepts any commit range specified in double-dot or
         triple-dot syntax and same predefined values are available as stated
-        above in the description of <b>branch_range</b> property.  
+        above in the description of <b>branch_range</b> property.
+      </td>
+    </tr>
+    <tr>
+      <td> exclude </td>
+      <td>
+        List of file patterns to exclude from the change-in pattern match.
+        For example, `change_in('/', exclude: ['/docs'])` will evaluate to true
```

Comment:
The `exclude` needs to be within the map, e.g. `change_in('/', {exclude: ['/docs']})`